config/morse/signed_aat.go

```diff
@@ -3,8 +3,8 @@ package morse
 import (
 	"fmt"
 
+	"github.com/buildwithgrove/path/config/utils"
```

Comment:
Could you please update all imports so self-imports are at the bottom?

Reply:
Sure.

Reply:
> Could you please update all imports so self-imports are at the bottom?

Done.